docs/release-notes.rst

```diff
@@ -1,3 +1,289 @@
+
+------
+
+**Visibility table status icons render few seconds after data for scale configs**
+
+**Workaround:** None
+
+------
+
+**BIG-IP v12 displays BIG-IP logs in local timezone**
+
+**Visibility Dashboard → View Logs** (for both VIP and Node) will display logs in UTC. But for BIG-IP v12, the logs are displayed in local timezone (timezone of the BIG-IP).
+
+**Workaround:** Upgrade BIG-IP to v13 or higher.
+
+------
+
+**Visibility Dashboard filter may display additional logs with interface filter**
+
+**Visibility Dashboard → BIG-IP Endpoint Details → Interface → View Logs** (for BIG-IP interfaces) displays interface logs. By default it applies the filter of interface name (for example: 1.1 or 1.2); and hence only the logs with interface name in them are displayed. This filtering logic may not work as expected and display additional logs which have interface names as a substring (For example: **1.2** is a substring in the log "Pool /Partition/Application/web_pool member **/Partition/12.14.1.2:8080** monitor status down").
+
+**Workaround:** None.
+
+------
+
+L4-L7 App Services
+------------------
+
+**Application services declaration not deleted**
+
+If your AS3 declaration contains “optimisticLockKey” mentioned explicitly, the Application Services configuration may not be deleted completely, even after multiple attempts from the application UI. However, the configuration is removed from the BIG-IP device.
+
+**Workaround**: Upload an AS3 sample declaration to the app and then perform a :guilabel:`Delete all` operation. (Use :guilabel:`View AS3 Declaration` and click :guilabel:`Delete`.)
+
+-------
+
+**Once all Pool Member cards are removed from the template form, adding a new pool member card fails on FASC FAST templates UI**
+
+If a user clicks the "-" sign in front of the pool members card and tries to add pool member card again by clicking the "+" icon, then the pool member card is not populated in template form.
+
+**Workaround:** Refresh the basic subtab on the L4-L7 tab
+
+------
+
+
+**A Text input field is wrongly displayed for 'Notice: Beta Test' in 'bigip-fast-templates/microsoft adfs' and 'bigip-fast-templates/microsoft exchange' FAST templates**
+
+'bigip-fast-templates/microsoft adfs' and 'bigip-fast-templates/microsoft exchange' FAST templates display a text input against the 'Notice: Beta Test' field. An error will be seen if this entry is filled during the form submission.
+
+**Workaround:** Leave the 'Notice: Beta Test' field blank during form submission.
+
+------
+
+
+Dynamic Endpoint Attach Detach
+------------------------------
+
+
+**Error on EPG mapping delete operation**
+
+When a dynamic endpoint mapping is added to an application using **Manage Endpoint Mappings**, the application is created on the BIG-IP. If this mapping is deleted using the **RESET** button on **Manage Endpoint Mappings** form, users may encounter an error "The requested Pool Member (/Partition/App/Pool /NodePartition/NodeIP) was not found."
```

Comment:
...may encounter an error such as "The requested...
or
...may encounter the error: "The requested...